src/somd2/runner/_dynamics.py

```diff
@@ -29,6 +29,11 @@
 
 from somd2 import _logger
 
+if _platform.system() == "Windows":
+    lam_sym = "lambda"
+else:
+    lam_sym = "λ"
+
```

Comment:
Can `lam_sym` be private, i.e. `_lam_sym`. The same is true in `_runner.py`. It might be better to just declare it in one of the files, then import in the other. That way if we make any changes then we don't need to update in two places.